frontend/app/tests/unit/specs/store/settings/frontend.spec.ts

```diff
@@ -64,6 +64,7 @@ describe('settings:frontend', () => {
       itemsPerPage: 10,
       amountRoundingMode: BigNumber.ROUND_UP,
       valueRoundingMode: BigNumber.ROUND_DOWN,
+      subscript_decimals: false,
```

Comment:
this line should also be removed here the other should be enough,
the undefined error is probably because of the accidental removal of the `language` from the store.